Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -184,6 +184,24 @@ public Thickness CodeBorderThickness
             typeof(MarkdownTextBlock),
             new PropertyMetadata(null, OnPropertyChangedStatic));
 
+        /// <summary>
+        /// Gets or sets the Prefix of Image Link.
+        /// </summary>
+        public string ImageLinkPrefix
```

Comment:
That would be a better name.

Reply:
The reason behind `ImageLinkPrefix` is because i want to use the same `Prefix` for actual links also instead of 2 different DP's.  While this is still not implemented, I want to implement this in such a way that even a relative link works like a normal link when `LinkClicked` event is called.

Reply:
@avknaidu you mean so that the developer doesn't have to add the detection and relative link parsing to their `LinkClicked` event handler?

In that case, I'd suggest `UriPrefix`.

Reply:
@michael-hawker . Done. I will push the commit shortly.